### PR TITLE
Scaling alert component fixes

### PR DESCRIPTION
**Pull request by CilliersN**
as per issue #570 
Components don't scale correctly on smaller devices.
Alert message box doesn't have a top margin -only bottom. This is a bootstrap component problem and as such, top margins need to be applied wherever these alerts are used.

Result:
Screenshot 2020-02-12 at 10 26 14

Errors were fixed throughout the repo. Indentation Cleanup was also done on files that were worked on

### CODE REVIEW

**Comment by CilliersN** [comment]
Please review but do not merge yet, I just want to confirm marginal changes on alerts

File: webapp/src/pages/createAccount/CreateAccount.css [hunk 1]
```diff
@@ -2,4 +2,8 @@
   .CreateAccount {
     padding: 60px 0;
   }
+}
+
+.alert-container{
```
Comment (avishkar58) [inline webapp/src/pages/createAccount/CreateAccount.css]: I think you can get away with defining this style once in App.css, which is available to every page anyway.

**Comment by avishkar58** [comment]
By the way @CilliersN, until #567 lands, if you change the domain and system_url of EEML in the organisation table on your local DB to "localhost", you can see the event home page, application form etc - I'm sure there are a lot more styling issues there!

File: webapp/src/pages/createAccount/CreateAccount.css (hunk 1 above)
Comment (CilliersN) [reply webapp/src/pages/createAccount/CreateAccount.css]: ahhh yes, thats right. 
Sorry dont know how ive missed that :)
still finding my way around your repo structure

**Comment by CilliersN** [comment]
> By the way @CilliersN, until #567 lands, if you change the domain and system_url of EEML in the organisation table on your local DB to "localhost", you can see the event home page, application form etc - I'm sure there are a lot more styling issues there!

Thanx for this... will check it out.

File: webapp/src/pages/createAccount/CreateAccount.css (hunk 1 above)
Comment (CilliersN) [reply webapp/src/pages/createAccount/CreateAccount.css]: I've implemented the new changes.

**Pull request merged**